pkg/profiler/cpu/maps.go

```diff
@@ -151,6 +151,7 @@ var (
 	errUnrecoverable             = errors.New("unrecoverable error")
 	errTooManyExecutableMappings = errors.New("too many executable mappings")
 	ErrNeedMoreProfilingRounds   = errors.New("not enough profiling rounds with this unwind info")
+	ErrTextSectionNotFound       = errors.New("could not find .text section")
```

Comment:
We don't actually return this error anywhere so the `errors.Is` above won't work.

Reply:
And if this doesn't cross the package boundary, maybe we should consider not exporting it?

Reply:
agreed! 
it was a typo on my end, but thanks so much for catching it!